ui.go

```diff
@@ -219,7 +225,7 @@ func (e *editor) refresh() {
 			e.currentDeviceConfig.Pages[e.currentDevice.Page] = append(e.currentDeviceConfig.Pages[e.currentDevice.Page], api.Key{})
 		}
 		b.(*button).key = e.currentDeviceConfig.Pages[e.currentDevice.Page][b.(*button).keyID]
-		b.Refresh()
+		go b.Refresh()
```

Comment:
This is a warning to me - do you know why it was necessary? The Refresh code runs on a different event loop already, adding this means you're likely to take 2 frames to draw instead of 1

Reply:
I thought it was necessary to fix the issue with crashing on editor refresh, seems not, it's gone now